lua/graph.lua

```diff
@@ -121,9 +127,7 @@ sidewalk = {
 ["this"] = "true",
 ["detached"] = "false",
 ["raised"] = "true",
-["separate_double"] = "false",
-["sidepath"] = "false",
```

Comment:
@CuriOusQuOkka why was sidepath and explicit removed?

Reply:
It was an accident, fixed. Thanks